src/app/technical-record/adr/adr-details/edit/adr-details-edit.component.html

```diff
@@ -74,7 +74,7 @@ <h2 class="govuk-heading-m">ADR details</h2>
 
           <ng-template #compatibilityGroup>
             <div class="govuk-form-group govuk-!-margin-left-3">
-              <h4 class="govuk-heading-m">Compatibility group J</h4>
+              <h4 class="govuk-heading-m">Compatibility group J(optional)</h4>
```

Comment:
yes I agree with @JonLazariniDD . @rspasici it should be like olu (optional) . Please lets update all where optional is used